app/helpers/candidate_details_helper.rb

```diff
@@ -0,0 +1,5 @@
+module CandidateDetailsHelper
+  def includes_eu_eea_swiss?(nationalities)
+    EU_EEA_SWISS_COUNTRY_CODES.intersect?(nationalities.map { |name| NATIONALITIES_BY_NAME[name] })
+  end
+end
```

Comment:
@JR-G  @inulty-dfe I wanna hear your opinions about helpers and form objects. Should we have a concerns folder in app/forms or move everything to app/helpers? @avinhurry can you investigate some other modules that is used in app/forms that were moved to app/helpers please?

Reply:
Looks like `Nationalities` is a module.

A look up interface would be useful here. We'd be able to enquire it's methods rather than them all being unrelated constants in `config/initializers/nationalities.rb`

Reply:
Let's consider this as a separate piece of work as this PR is already quite big and we could look at using a lookup for the other constants in nationalities.